packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -2754,13 +2754,13 @@ function updateDehydratedSuspenseComponent(
   }
 }
 
-function scheduleWorkOnFiber(fiber: Fiber, renderLanes: Lanes) {
+function scheduleContextWorkOnFiber(fiber: Fiber, renderLanes: Lanes) {
   fiber.lanes = mergeLanes(fiber.lanes, renderLanes);
   const alternate = fiber.alternate;
   if (alternate !== null) {
     alternate.lanes = mergeLanes(alternate.lanes, renderLanes);
   }
-  scheduleWorkOnParentPath(fiber.return, renderLanes);
+  scheduleContextWorkOnParentPath(fiber.return, renderLanes, null);
```

Comment:
Is it bad we're not passing any propagation root here? So for Suspense changes, we'll always traverse to the actual root now whereas previously we didn't have to. Should we pass something? Maybe `fiber.return.return`?

Reply:
I pushed another commit that specifies `fiber.return.return` for this case. Seems like that would be enough.